language/move-prover/src/spec_translator.rs

```diff
@@ -1890,15 +1902,15 @@ impl<'env> SpecTranslator<'env> {
             return self.translate_exp(exp);
         }
         let loc = self.module_env().env.get_node_loc(node_id);
-        if let Some((name, binding)) = self.get_decl_var(&loc, vars) {
-            let name_str = self.module_env().symbol_pool().string(name);
+        if let [var] = vars {
```

Comment:
Did not know this is possible!

Reply:
Me neither, I found it by reading this file! Slice patterns were added in Rust 1.26 apparently.